### ISSUE TITLE
Lambda functions with nodejs14.x

### ISSUE THREAD

**Issue by rvanbutselaar** [awslabs/landing-zone-accelerator-on-aws#240]
**Describe the bug**
We are receiving security alerts from Trusted Advisor.
Some Lambda functions deployed by this solution are using nodejs14, which will be deprecated within 120 days. 
The Lambda functions running on nodejs14 are all starting with the following description: "AWS CDK resource provider framework - ...".
It looks like these Lambda's are generated by AWS CDK custom resources. AWS CDK is using a more recent version of nodejs for custom resources in the most recent version of CDK: https://github.com/aws/aws-cdk/releases/tag/v2.92.0.

**To Reproduce**
Deploy / upgrade to version [v1.4.3](https://github.com/awslabs/landing-zone-accelerator-on-aws/releases/tag/v1.4.3)

**Expected behavior**
Use a more recent version of nodejs: nodejs16 or nodejs18

**Please complete the following information about the solution:**

- [ ] Version: v1.4.3
- [ ] Region: eu-central-1
- [ ] Was the solution modified from the version published on this repository?
- [ ] If the answer to the previous question was yes, are the changes available on GitHub?
- [ ] Have you checked your [service quotas](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html) for the sevices this solution uses?
- [ ] Were there any errors in the CloudWatch Logs?

**Screenshots**
If applicable, add screenshots to help explain your problem (please **DO NOT include sensitive information**).

**Additional context**
```
$ aws lambda list-functions |jq -r '.Functions[] |  select(.Runtime | startswith("nodejs14.x")) | .FunctionName, .Runtime'
AWSAccelerator-PrepareSta-CreateCTAccountsCreateCo
nodejs14.x
AWSAccelerator-PrepareSta-CreateCTAccountsCreateCo
nodejs14.x
AWSAccelerator-LoggingSta-AWSServiceRoleForAWSClou
nodejs14.x
AWSAccelerator-PrepareSta-CreateOrganizationAccoun
nodejs14.x
AWSAccelerator-PipelineSt-PipelineAWSServiceRoleFo
nodejs14.x
AWSAccelerator-PrepareSta-CreateOrganizationAccoun
nodejs14.x
AWSAccelerator-LoggingSta-AutoScalingServiceLinked
nodejs14.x
AWSAccelerator-PrepareSta-CreateCTAccountsCreateCo
nodejs14.x
AWSAccelerator-Organizati-IdentityCenterAdminframe
nodejs14.x
AWSAccelerator-PrepareSta-ValidateEnvironmentConfi
nodejs14.x
AWSAccelerator-PrepareSta-CreateOrganizationAccoun
nodejs14.x
```

**Comment by bo1984** [comment]
@rvanbutselaar thank you for bringing this to our attention, right now we have an issue already tracked for this. I will leave this issue open in case you have additional questions or concerns, and we'll update you once this issue has been addressed.

**Comment by devalibvr** [comment]
Across all accounts, we received the same alert. Is there anything we need to do or will the LZA team resolve it in future updates?

**Comment by joshuahigginson1** [comment]
This is the same with our team, so +1 ^.^

**Comment by andy-nyunt** [comment]
+1
I'm guessing it's affecting all LZA deployments
Be good to understand if  future relase of LZA will address this or if users of LZA needs to action this directly

**Comment by awsclemj** [comment]
Hello,

A fix for this issue has been included in our [v1.5.0 release](https://github.com/awslabs/landing-zone-accelerator-on-aws/releases/tag/v1.5.0). I will be closing this issue as resolved. Thanks for your support of Landing Zone Accelerator!

**Comment by frankscalzo** [comment]
> Hello,
> 
> A fix for this issue has been included in our [v1.5.0 release](https://github.com/awslabs/landing-zone-accelerator-on-aws/releases/tag/v1.5.0). I will be closing this issue as resolved. Thanks for your support of Landing Zone Accelerator!

@awsclemj  even after i have updated to and i am currently on 1.5.2 i am still seeing Node 14 and 16 through out my deployment is there a way to force the update

**Comment by awsclemj** [comment]
Hi @frankscalzo, Node 16 is the expected runtime. We will be updating to Node 18 in a subsequent release prior to its end of life support. 

If you still have Node 14 in your environment after the upgrade, it may be custom config rules you have defined in your security config. You can upgrade the runtime of those in the runtime property of [CustomRuleConfig](https://awslabs.github.io/landing-zone-accelerator-on-aws/classes/_aws_accelerator_config.ConfigRule.html#customRule).

I hope this helps!
